packages/opencensus-exporter-stackdriver/test/test-stackdriver-monitoring-utils.ts

```diff
@@ -616,4 +618,34 @@ describe('Stackdriver Stats Exporter Utils', () => {
       });
     });
   });
+
+  describe('partitionList()', () => {
+    const timeSeriesList: TimeSeries[] = [];
+    for (let i = 0; i < 205; i++) {
+      timeSeriesList.push({
+        metric: {
+          type: 'custom.googleapis.com/opencensus/metric-name-' + i,
+          labels: {},
+        },
+        resource: { type: 'global', labels: {} },
+        metricKind: MetricKind.GAUGE,
+        valueType: ValueType.INT64,
+        points: [
+          {
+            interval: { endTime: '2019-01-09T01:52:55.00000001Z' },
+            value: {
+              int64Value: i,
+            },
+          },
+        ],
+      });
+    }
+
+    it('should return a partition lists', () => {
+      const list = partitionList(timeSeriesList, 200);
```

Comment:
Would it make sense to test the exporter directly rather than the helper function? That would enable check that the helper function is used correctly and would implicitly check the helper (which could then even be made non-exported).

I'm open to leaving as-is for now if you prefer though.